pages/linux/getfacl.md

```diff
@@ -0,0 +1,15 @@
+# getfacl
+
+> Get file access control lists.
+
+- Display the file access control list:
+
+`getfacl -a {{path/to/file_or_folder}}`
```

Comment:
This seems to give the same result as without `-a`. Let's remove it in that case.

Reply:
Yes, you're right. The default operation of `getfacl` is `getfacl -a`

Reply:
But I don't think `getfacl -PR` will be useful for most situations. The output format of `getfacl -PR` is not as good as `ls -al`.
So I think `getfacl` is more suit for single file.

Reply:
Understood. Let's do the first part then.